ci/deps/actions-310-numpydev.yaml

```diff
@@ -6,11 +6,16 @@ dependencies:
 
   # build dependencies
   - versioneer[toml]
+  - meson[ninja]=1.0.1
+  - meson-python=0.13.0
 
   # test dependencies
   - pytest>=7.0.0
   - pytest-cov
-  - pytest-xdist>=2.2.0
+  # Once pytest-cov > 4 comes out, unpin this
+  # Right now, a DeprecationWarning related to rsyncdir
```

Comment:
Is this still an issue?

Reply:
Yes, pytest-cov hasn't released anything since September sadly.

Reply:
Hmm do you know why we are seeing this DeprecationWarning which changing to meson? (just curious, not a blocker)